ci/requirements-3.7-0.24.2.conda

```diff
@@ -1,14 +1,17 @@
 codecov
```

Comment:
Pinning dependencies was causing a bunch of problems for the CircleCI (3.7) build. We should move these tests over to Kokoro